docs/SUMMARY.md

```diff
@@ -18,6 +18,7 @@
     * [Lambda Functions](workflows/lambda.md)
     * [Dockerization](workflows/docker.md)
   * [Workflow Configuration How-to's](workflows/workflow-configuration-how-to.md)
+    * [Configuring Retries of Steps](workflows/workflow-retry-configuration.md)
```

Comment:
I'm thinking instead of here we should just move this underneath of "Error Handling" in the data cookbook.